error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

Hello,

I have followed your instructions, and at the last step:
(System is Ubuntu x64 from a fresh install)

``` python
(leapcast)cve@CVE:~/leapcast$ pip install .
Unpacking /home/cve/leapcast
  Running setup.py egg_info for package from file:///home/cve/leapcast

Requirement already satisfied (use --upgrade to upgrade): distribute in /home/cve/.virtualenvs/leapcast/lib/python2.7/site-packages/distribute-0.6.34-py2.7.egg (from Leapcast==0.0.1)
Downloading/unpacking twisted (from Leapcast==0.0.1)
  Running setup.py egg_info for package twisted

Downloading/unpacking tornado (from Leapcast==0.0.1)
  Running setup.py egg_info for package tornado

    warning: no previously-included files matching '_auto2to3*' found anywhere in distribution
Downloading/unpacking zope.interface>=3.6.0 (from twisted->Leapcast==0.0.1)
  Running setup.py egg_info for package zope.interface

    warning: no previously-included files matching '*.dll' found anywhere in distribution
    warning: no previously-included files matching '*.pyc' found anywhere in distribution
    warning: no previously-included files matching '*.pyo' found anywhere in distribution
    warning: no previously-included files matching '*.so' found anywhere in distribution
Installing collected packages: twisted, tornado, Leapcast, zope.interface
  Running setup.py install for twisted

    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c conftest.c -o conftest.o
    building 'twisted.runner.portmap' extension
    x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c twisted/runner/portmap.c -o build/temp.linux-x86_64-2.7/twisted/runner/portmap.o
    twisted/runner/portmap.c:10:20: fatal error: Python.h: No such file or directory
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    Complete output from command /home/cve/.virtualenvs/leapcast/bin/python -c "import setuptools;__file__='/home/cve/.virtualenvs/leapcast/build/twisted/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Xw3x6q-record/install-record.txt --single-version-externally-managed --install-headers /home/cve/.virtualenvs/leapcast/include/site/python2.7:
    running install

running build

running build_py

running egg_info

writing requirements to Twisted.egg-info/requires.txt

writing Twisted.egg-info/PKG-INFO

writing top-level names to Twisted.egg-info/top_level.txt

writing dependency_links to Twisted.egg-info/dependency_links.txt

warning: manifest_maker: standard file '-c' not found



reading manifest file 'Twisted.egg-info/SOURCES.txt'

writing manifest file 'Twisted.egg-info/SOURCES.txt'

running build_ext

x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c conftest.c -o conftest.o

building 'twisted.runner.portmap' extension

x86_64-linux-gnu-gcc -pthread -fno-strict-aliasing -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -fPIC -I/usr/include/python2.7 -c twisted/runner/portmap.c -o build/temp.linux-x86_64-2.7/twisted/runner/portmap.o

twisted/runner/portmap.c:10:20: fatal error: Python.h: No such file or directory

compilation terminated.

error: command 'x86_64-linux-gnu-gcc' failed with exit status 1

----------------------------------------
Command /home/cve/.virtualenvs/leapcast/bin/python -c "import setuptools;__file__='/home/cve/.virtualenvs/leapcast/build/twisted/setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-Xw3x6q-record/install-record.txt --single-version-externally-managed --install-headers /home/cve/.virtualenvs/leapcast/include/site/python2.7 failed with error code 1 in /home/cve/.virtualenvs/leapcast/build/twisted
Storing complete log in /tmp/tmpKBSTMP
```

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ Clone this directory, then run ```python setup.py develop```
 ```
 git clone https://github.com/dz0ny/leapcast.git
 cd ./leapcast
-sudo apt-get install virtualenvwrapper python-pip
+sudo apt-get install virtualenvwrapper python-pip python-twisted-web python2.7-dev
 mkvirtualenv leapcast
 pip install .
 ```

